### DISCUSSION TITLE
Looking for new c++ maintainer

### DISCUSSION THREAD

**Discussion by borgmanJeremy** [flameshot-org/flameshot#2961] [General]
Hello Everyone,

I wanted to inform the Flameshot community that effective immediately I am stepping away from Flameshot development. My life outside of Flameshot has gotten very busy and I am having a hard time managing all my competing priorities.  

I really have had a blast working on this for the last few years and want to leave on a positive note rather than continuing to wear down and slowly ghost issues.

This doesn't mean the end of Flameshot. The entire time I have worked as a developer on the project @mmahmoudian has been our community organizer and main bug triager. He has full access to all the Flameshot assets and accounts and will be able to grant access to a new maintainer if one comes along.

Thanks for all the community support the last few years.

**Comment by MaybeThisIsRu** [comment]
Thanks for you work! 💛😊

**Comment by DsChauhan08** [comment]
Are you looking for any web developer or web designer?

**Reply by mmahmoudian** [reply]
Of course, any contribution is welcome. Our website's repo is in the following link:

https://github.com/flameshot-org/flameshot-org.github.io

And this it how it translates to website:
https://flameshot.org

There are few improvements that would be great to have:
- multilingual support
- having the https://flameshot.org/docs index page to have better style (e.g adding icons) and hierarchy
- vertical hamburger menu for mobile view (at the moment it is horizontal)
- reducing the javascript (would be great if the website works without any JS as well)
- embedding some peertube videos (we should also make some videos to showcase features and for introduction)
